### ISSUE TITLE
Increase Accessibility

### ISSUE THREAD

**Issue by goibon** [goibon/spooky-meter#8]
Everyone should have equal access to check the current spooky level and as such, we should make sure to follow best practices for accessibility on the web. Some points that could be improved are:
- Use proper [alt] attributes on `img` tags
- Specify language in the `html` tag
